ntc_templates/templates/zyxel_os_cfg_lan_get.textfsm

```diff
@@ -0,0 +1,15 @@
+Value NAME (.+?)
+Value IP (((\d+\.){3}\d+)?)
```

Comment:
```suggestion
Value IP ((\d+\.){3}\d+)
```

Reply:
I think this works?? I don't think we should check for IP or nothing here. We should also standardize on a way to match an IP. 

@jvanderaa @jmcgill298 any thoughts?

Reply:
I do agree. And I was also thinking that I was using different ways to describe IPs. A standard would be welcome